pdsfile/pds3file/rules/COISS_xxxx.py

```
@@ -3,6 +3,7 @@
 ##########################################################################################
 
 import pdsfile.pds3file as pds3file
+import pdsfile.pds4file as pds4file
```

Ruff (0.15.5)

[warning] Use `from pdsfile import pds4file` in lieu of alias

Replace with `from pdsfile import pds4file`

(PLR0402)